_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,97 @@ nav_order: 50
-- Must have a valid time-based X-axis
+- The visualization must be a [Vizlib line chart](https://community.vizlib.com/support/solutions/articles/35000107262-vizlib-line-chart-introduction).
+- The visualization must contain at least a Y-axis metric aggregation.
+- The visualization must not have non-Y-axis metric aggregation types.
+- The visualization must use the date histogram aggregation type for the X-axis bucket.
+- The visualization must have an X-axis on the bottom.
+- The visualization must define one X-axis aggregation bucket.
+- The visualization must have a valid time-based X-axis.
 
-## Creating alerting monitors
+## Creating alerting monitors from within visualizations
 
-By default, when you begin to create the alert monitor workflow using the Dashboard interface, you are presented with a menu-driven interface. This interface provides a range of options that are displayed in full screen, in pop-ups, in pull-downs, or in dropdowns. They allow you to define the metrics that can be monitored, set thresholds, customize triggers that automate workflows, and generate actions when conditions are met. Currently, you can only create [per query monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/monitors/).
+By default, when you begin to create an alert monitor workflow using OpenSearch Dashboards, you are presented with a menu-driven interface. This interface provides a range of options that are displayed in full screen, in pop-ups, in pull-downs, or in dropdowns. These options are used to define the metrics that can be monitored, set thresholds, customize triggers that automate workflows, and generate actions when conditions are met. 
 
-To create an alerting monitor: 
+You can only create [per query and per bucket monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/per-query-bucket-monitors/) from within visualizations.
+{: .note}
 
-1. Choose **Dashboard** from the OpenSearch Dashboards main menu.
-2. From the **Dashboards** window, select **Create** and then choose **Dashboard**.
-3. Select **Add an existing**, then select the appropriate alerting visualization from the **Add panels** list. The visualization is added to the dashboard.
-4. From the visualization panel, choose the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}). 
-5. From the **Options** menu, select **Add alerting monitor**.
-6. Input information for **Monitor details** and **Triggers**.
-7. Choose **Create monitor**. The monitor is added to the visualization.  
+To create a monitor from within a visualization, follow these steps:
 
-An example of these steps is shown in the following screenshot.
+1. Save your dashboard. If you are unfamiliar with saving dashboards, see [Saving dashboards]({{site.url}}{{site.baseurl}}/dashboards/dashboard/index/#saving-dashboards).
+2. Select **Alerting** from the **Options** dropdown menu.
+3. 
 
```

Comment:
SME: Do you have a test environment I can use to write a tutorial. I'm unable to test these steps through my local or OpenSearch Playground.

Reply:
yes, can provide an instance of OpenSearch 2.14 to test the changes